supplementary_style_guide/glossary_terms_conventions/product_conventions/openshift.adoc

```diff
@@ -18,17 +18,23 @@
 *Description*: API objects are the resources that can be exposed by an API server
 at an endpoint for interrogation.
 
-Use initial capitalization and camel case for Kubernetes or OpenShift API
-objects and do not mark them up unless referring to a specific field or variable
-name from a spec or manifest file.
+When referring to terms that are API objects, if it is important to make the distinction that the context is in reference to the actual API object (like when defining or editing an object), use the proper PascalCase `Kind` for the object, and mark it up in monospace. For example:
```

Comment:
I would suggest revising this initial sentence to be more directly prescriptive: "When referring to an API object that shares its name with a common term, use the proper PascalCase `Kind` for the object, and mark it up in monospace. For example:"

Reply:
Thanks! I've reordered (because we want people using the general reference guidance if possible) and clarified a bit so it's not so long winded.